Microsoft.Toolkit.Uwp.UI/Converters/DateTimeToStringConverter.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ******************************************************************
```

Comment:
The good thing about  binding is that view just consumes what viewmodel exposes. yes often it is painful.. we have come a long way from WPF (in ways).. XAML is somewhat different plus default patterns is MVVM which tends to take a lot of load off the Views.

Since data is being bound to controls, i'd imagine most work is being done on UI thread. You dont really need StringFormat to run on UI thread.